docs/products/kafka/kafka-connect/howto/redis-streamreactor-sink.rst

```diff
@@ -0,0 +1,161 @@
+Create a Redis™* stream reactor sink connector by Lenses.io
+===========================================================
+
+The Redis stream reactor sink connector enables you to move data from an Aiven for Apache Kafka® cluster to a Redis™* database. The Lenses.io implementation enables you to write `KCQL transformations <https://docs.lenses.io/connectors/sink/redis.html>`_ on the topic data before sending it to the Redis database.
+
+
+.. _connect_redis_lenses_sink_prereq:
+
+Prerequisites
+-------------
+
+To setup a Redis sink connector, you need an Aiven for Apache Kafka service :doc:`with Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`.
+
+Furthermore you need to collect the following information about the target Redis database upfront:
+
+* ``REDIS_HOSTNAME``: The Redis hostname
+* ``REDIS_PORT``: the Redis port
+* ``REDIS_PASSWORD``: The Redis password
+* ``REDIS_SSL``: The Redis SSL setting
```

Comment:
You can add here that it should be either true or false. 
Although, for this particular case our error message was quite helpful :)